controller/transfer/project_transfer_manager.py

```diff
@@ -82,19 +82,19 @@ def ensure_project_exists(project_id: str) -> None:
 
 
 def import_sample_project(
-    user_id: str, organization_id: str, project_name: str
+    user_id: str, organization_id: str, project_name: str, project_type: str
 ) -> Project:
-    if project_name == "Clickbait - initial":
+    if project_type == enums.SampleProjectType.CLICKBAIT_INITIAL.value:
         file_name = "sample_projects/clickbait_initial.zip"
-    elif project_name == "Clickbait":
+    elif project_type == enums.SampleProjectType.CLICKBAIT.value:
         file_name = "sample_projects/clickbait.zip"
-    elif project_name == "AG News - initial":
+    elif project_type == enums.SampleProjectType.AG_NEWS_INITIAL.value:
         file_name = "sample_projects/ag_news_initial.zip"
-    elif project_name == "AG News":
+    elif project_type == enums.SampleProjectType.AG_NEWS.value:
         file_name = "sample_projects/ag_news.zip"
-    elif project_name == "Conversational AI - initial":
+    elif project_type == enums.SampleProjectType.CONVERSATIONAL_AI_INITIAL.value:
         file_name = "sample_projects/conversational_ai_initial.zip"
-    elif project_name == "Conversational AI":
+    elif project_type == enums.SampleProjectType.CONVERSATIONAL_AI.value:
         file_name = "sample_projects/conversational_ai.zip"
     else:
         raise Exception("Unknown sample project")
```

Comment:
Maybe it could make sense to include the given project_type in the exception message to give more information for a dev in case of an error and make bug tracking easier. I think there is all the time a conflict between reducing the information displayed to a minimum but still supporting the maintainability of an application as much as possible. But I am in support of including at least the essential information about why an error occurred into the message

Reply:
yes, it would make sense to include it